src/VNC.cpp

```diff
@@ -103,11 +96,7 @@ void arduinoVNC::begin(char *_host, uint16_t _port, bool _onlyFullUpdate) {
     opt.client.bpp = 16;
     opt.client.depth = 16;
 
-#ifdef ESP32
-    opt.client.bigendian = 0;

```

Comment:
is this tested on ESP8266 and ESP32?

Reply:
Well endianness seems to be a bigger topic. Actually both ESP8266 and ESP32 are little endian, so there shouldn't not be a difference. It tested ALL codecs with the ESP32 and more sure they work correctly with the same endianness.
Even though the little endian would be "correct" for the ESP32, on average it seems to be way easier to use big endian, because it removes the need to invert bigger buffers (like in the raw or zlib case). For the other cases it didn't seem like much overhead to swap two more bytes in some cases (like with RRE).

I was wondering if it makes sense to send the bigendian flag inverted to the server (so that for the ESPs it could be configured as 0 as one would expect, but sent as 1 to server, as the rest of the code expects), but in the end I was not sure this would be less confusing.